sass/_assets.scss

```diff
@@ -0,0 +1,42 @@
+.assets {
+    display: flex;
+
+    .card-list {
+        display: flex;
+        flex-wrap: wrap;
+    }
+
+    .card {
+        width: 30%;
+        margin-right: $card-list-gap;
+        height: initial;
+    }
+
+    .card-title {
+        font-size: 1.3rem
+    }
+
+    .card-description {
+        font-size: 1rem;
+        min-height: 4rem;
+        height: initial;
+    }
+
+    .card-image > img {
+        display: block;
+        height: auto;
+        max-width: 100%;
+    }
+
+    a {
+        text-decoration: none;
+    }
+
+    .book-content {
+        h3 {
+            font-size: 2.2rem;
```

Comment:
why did you add this one? it makes the `h3` very close to `h1` (no difference to my eye, `h1` is at 2.4rem)

Reply:
I'm open to downsizing a little bit, but imo the "default" set for the book is "too small" for sub-categories in this context. I'm definitely open to downsizing a little bit because I agree that its probably too close to the parent size.

Reply:
reduced the font a little, and added a ">" before headers. not sure about it though. What do you think?
<img width="928" alt="Screenshot 2021-05-27 at 22 47 41" src="https://user-images.githubusercontent.com/8672791/119895035-9f36a180-bf3d-11eb-964a-e68af27f6923.png">

Reply:
It definitely helps distinguish sub categories from categories, but I think it draws the eye a little bit too much and messes with alignment a bit too much. I think I'd prefer just bumping down the size another `0.1rem` or so.